src/htmlize/main.ml

```diff
@@ -48,9 +48,18 @@ let htmlize filename content =
   let ext = String.lowercase_ascii ext in
   match ext with
   | "md" ->
-    let content = try Omd.of_string content |> Patchtml.handle_file |> Omd.to_html with _ -> content in
-    Printf.bprintf b {|%s|} content;
-    Buffer.contents b
+    begin
+      match
+        Omd.of_string content |> Patchtml.handle_file |> Omd.to_html
+      with
+      | exception e ->
+        Format.eprintf "%s" (Printexc.to_string e);
```

Comment:
Launch this on a full switch, then check if there's some exceptions.
If yes, only catch these exceptions. Otherwise, remove this catch.